packages/camera/camera_avfoundation/ios/Classes/FLTCam_Test.h

```diff
@@ -5,6 +5,37 @@
 #import "FLTCam.h"
 #import "FLTSavePhotoDelegate.h"
 
+/**
+ A block type definition named VideoDimensionsForFormatBlock.
+
+ This block is intended to be used for determining the video dimensions (width and height) for a
```

Comment:
"This block is intended to be used for" adds a lot of unnecessary verbosity; people understand that the purpose of a declaration comment is to explain what something is used for.